src/test/java/org/openmrs/contrib/qaframework/automation/InpatienttSteps.java

```diff
@@ -57,9 +57,8 @@ public void systemloadsPatientVisitDashboardPage() {
 		assertTrue(visitsDashboardPage.containsText("Visits"));
 	}
 
-	// Admission encounter
 	@When("a user clicks on Admit to Inpatient button and selects location")
-	public void goToAdmitToInpatientPage() {
```

Comment:
Seems you are messing up with things here. This is the right way of naming the method

Reply:
> Seems you are messing up with things here

Have you had a glimpse at @dkayiwa 's comment [here](https://github.com/openmrs/openmrs-contrib-qaframework/pull/81#discussion_r695641883) ? gotToInpatientPage() is accomplishing Admission encounter in the nutshell and makes meaning to me from the technical angle to rename it to `admissionEncounter()` so as to eliminate un necessary comment(s) within our code base.